src/sql/parts/notebook/models/notebookContexts.ts

```diff
@@ -54,7 +54,7 @@ export class NotebookContexts {
 	public static async getContextsForKernel(connectionService: IConnectionManagementService, connProviderIds: string[], kernelChangedArgs?: nb.IKernelChangedArgs, profile?: IConnectionProfile): Promise<IDefaultConnection> {
 		let connections: IDefaultConnection = this.DefaultContext;
 		if (!profile) {
-			if (!kernelChangedArgs || !kernelChangedArgs.newValue ||
+		if (!kernelChangedArgs || !kernelChangedArgs.newValue ||
```

Comment:
Seems like the only change here is you accidentally removed a tab that should still be there. Can you add this back in? And the newline at end of file?